### PR TITLE
Fix/tbd 2/css fix

### PR DESCRIPTION
**Pull request by Yury-Razhkou**
Related to: https://oat-sa.atlassian.net/browse/TBD-2
Ways to test:
Open `TAO_URL/ltiDeliveryProvider/DeliveryRunner/runDeliveryExecution` back button will appears only if HTTP_REFERER is the same as TAO_URL.

### CODE REVIEW

File: views/templates/error/error500.tpl [hunk 1]
```diff
@@ -10,7 +10,7 @@
 </head>
 
 <body>
-
+<?php xdebug_break();?>
```
Comment (Alroniks) [inline views/templates/error/error500.tpl]: Seems it should be removed from production code

File: models/classes/mvc/error/HtmlResponse.php [hunk 2]
```diff
@@ -29,15 +29,19 @@
...
+
     public function send()
     {
         if (DEBUG_MODE) {
             $message = $this->exception->getMessage();
             $trace = $this->exception->getTraceAsString();
         }
+        $returnUrl = $_SERVER['HTTP_REFERER'];
+        if (parse_url($returnUrl, PHP_URL_HOST) != parse_url(ROOT_URL, PHP_URL_HOST)) {
+            $returnUrl = false;
```
Comment (Alroniks) [inline models/classes/mvc/error/HtmlResponse.php]: Where this var is used? It is not passed anywhere.

Comment (Yury-Razhkou) [reply models/classes/mvc/error/HtmlResponse.php]: It is the way how templates rendered here :`require Template::getTemplate('error/error'...`
This variable used in template:
```
<?php if (!empty($returnUrl)) : ?>
<li><a href="<?= $returnUrl ?>"><?=__('Go Back')?></a></li>
<?php endif; ?>
```

File: views/templates/error/error500.tpl (hunk 1 above)
Comment (Yury-Razhkou) [reply views/templates/error/error500.tpl]: definitely yes! thanks.

File: models/classes/mvc/error/HtmlResponse.php (hunk 2 above)
Comment (SergiiTao) [reply models/classes/mvc/error/HtmlResponse.php]: IMHO looks a bit more readable. but not important

```php
$referer = $_SERVER['HTTP_REFERER'];

$returnUrl = (parse_url($referer, PHP_URL_HOST) === parse_url(ROOT_URL, PHP_URL_HOST))
    ? htmlentities($referer, ENT_QUOTES)
    : false;
```

**Pull request merged**